src/Button/Button.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { StyledButton ,SecondButton, ThridButton, FourthButton, Primarybuttonhover,SecondButtonhover,ThridButtonhover,FourthButtonhover, Save_hover,Save_new,P1,P2,P3,P4} from './Button.Styled';
+export const Button= () => {
+  return (
+    <div>
+      {/* <button>Hello</button> */}
+      <StyledButton>PRIMARY BUTTON</StyledButton>
+      <SecondButton>SECONDARY BUTTON</SecondButton>
+      <ThridButton>SUCESSON BUTTON</ThridButton>
+      <FourthButton>WARNING BUTTON</FourthButton>
+    </div>
+  );
+};
+export const Hover=()=>{
+  return(
+    <div>
+      <Primarybuttonhover>PRIMARY BUTTON</Primarybuttonhover>
+      <SecondButtonhover>
+      SECONDARY BUTTON
+      </SecondButtonhover>
+      <ThridButtonhover>
+        SUCESSON
+      </ThridButtonhover>
+      <FourthButtonhover>
+        WARNING BUTTON
+      </FourthButtonhover>
+    </div>
+  );
+};
+export const Icon=()=>{
+  return(
+    <div>
+      <Save_hover>
+      <svg
```

Comment:
@KBhautik6 create svg file for this image and use here

Reply:
I try but it show some error so i directly use the svg again 

ERROR in ./src/image/logo.svg
Module build failed (from ./node_modules/@svgr/webpack/lib/index.js):
SyntaxError: unknown: Namespace tags are not supported by default. React's JSX doesn't support namespace tags. You can set `throwIfNamespace: false` to bypass this warning.